stable/mariadb/templates/svc.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.metrics.enabled }}
+  annotations:
+{{ toYaml .Values.metrics.annotations | indent 4 }}
+{{- end }}
 spec:
   type: {{ .Values.serviceType }}
   ports:
```

Comment:
I know you've addressed this question already, but is there harm in exposing port 9104 on the Service? It would still fulfill your goal, but would also make the metrics scrape-able by an external Prometheus if `serviceType=nodePort`.